pkg/autodiscover/autodiscover.go

```diff
@@ -175,6 +176,14 @@ func DoAutoDiscover(config *configuration.TestConfiguration) DiscoveredTestData
 		os.Exit(1)
 	}
 	data.Crds = FindTestCrdNames(config.CrdFilters)
+	clusterCrds, err := GetClusterCrdNames()
```

Comment:
We now have `GetClusterCrdNames()` and `FindTestCrdNames()`.  Would it have been possible to combine the logic?

Reply:
Done - the unit tests were a bit tricky though.